tests/ignite/metrics/test_confusion_matrix.py

```diff
@@ -577,3 +577,39 @@ def test_multinode_distrib_cpu(distributed_context_multi_node_gloo):
 def test_multinode_distrib_gpu(distributed_context_multi_node_nccl):
     device = "cuda:{}".format(distributed_context_multi_node_nccl['local_rank'])
     _test_distrib_multiclass_images(device)
+
+
+def test_dice_coefficient():
+    n_classes = 3
+    logits = [[np.random.random() for j in range(n_classes)] for i in range(5)]
```

Comment:
@PyExtreme please create directly 2D arrays as `np.random.rand(5, n_classes)` for logits and y_true